README.md

```diff
@@ -280,6 +280,49 @@ Please go the Remote Management tab of the Settings page and check which remote
 
 ![Remote Management Tab](./assets/screenshot-3.png)
 
+### RESTapi for Remote Management
+
+GitHub updater also supports other customized continous integration workflows, and it is possible to integrate with other services than those discussed above. For this, there is a RESTapi available for telling GitHub Updater to update themes and plugins to the latest version from their repositories.
+
+On the Remote Management tab, you will see a url that serves as the enpoint for this api. This url will look something like:
+
+    http://localhost/wordpress/wp-admin/admin-ajax.php?action=github-updater-update&key=76bb2b7c819c36ee37292b6978a4ad61
+
+The exact url will of course depend on your system. The part after the `key` attribute is automatically generated on the first activation of the GitHub updater plugin and is used for authentication. Any person or entity knowing this key will be able to change the versions of your installed plugins, but nothing else.
+
+Now, if we would use `curl` to access the url exactly like it is written on the Remote Management tab, we would see something like this:
+
+    $ curl "http://localhost/wordpress/wp-admin/admin-ajax.php?action=github-updater-update&key=76bb2b7c819c36ee37292b6978a4ad61"
+    {
+        "message": "No plugin or theme specified for update.",
+        "error": true
+    }
+
+This error message is given because GitHub updater requires us to specify either a theme or a plugin that we wish to update. This is specified using the `theme` or `plugin` attributes, and the theme or plugin is identified by its slug. Let's try to update a plugin:
+
+    $ curl "http://localhost/wordpress/wp-admin/admin-ajax.php?action=github-updater-update&key=76bb2b7c819c36ee37292b6978a4ad61&plugin=myplugin"
```

Comment:
Good point. Actually, the url was pointing to a repo that didn't even exist, I just thought of it as an example. I created a separate repo and pointed it to that one, so now it will work if people copy-paste from the docs. If you don't mind, I would like to keep it pointing to one of my repos. This is partially because I like to see my name there, and partially because of the street cred it would give me from all the hackers (or recruiters) who would see my name. However, you are the main author of the plugin, so if you would like to see your name there for the same reason, I have total respect for that.